packages/core/admin/admin/src/content-manager/hooks/useFetchContentTypeLayout/index.js

```diff
@@ -24,7 +23,7 @@ const useFetchContentTypeLayout = (contentTypeUID) => {
       dispatch({ type: 'GET_DATA' });
 
       try {
-        const endPoint = getRequestUrl(`content-types/${uid}/configuration`);
+        const endPoint = `/content-manager/content-types/${uid}/configuration`;
```

Comment:
Do you think we could use that opportunity to inline this into the `get` call? The casing sets me really off :D

Reply:
done